crates/matrix-sdk/src/sliding_sync/cache.rs

```diff
@@ -216,6 +208,27 @@ mod tests {
     use super::*;
     use crate::{Client, Result};
 
+    #[test]
+    fn test_cant_cache_without_a_storage_key() -> Result<()> {
```

Comment:
```suggestion
    fn test_cannot_cache_without_a_storage_key() -> Result<()> {
```

?

Reply:
Less risk of confusion of `can` with `cant`, I'll take it, thanks!